src/applications/vaos/reducers/expressCare.js

```diff
@@ -19,11 +22,13 @@ import { FETCH_STATUS } from '../utils/constants';
 
 const initialState = {
   windowsStatus: FETCH_STATUS.notStarted,
-  hasWindow: false,
-  allowRequests: false,
```

Comment:
I moved `allowRequests` because I need to save the windows with non-formatted dates, to get whatever facility has a currently active window. With that data saved in Redux, it's easy enough to just calculate it in a selector.